tests/Signals_perftest.m

```diff
@@ -152,7 +155,21 @@ function test_SignalOps(testCase, SignalOps)
             for i = 1:testCase.Reps
               post(testCase.A, s);
             end
-          end         
+          end
+        case {'delay'}
+          % special case: have to use `runSchedule`     
+          while (testCase.keepMeasuring)
+            for i = 1:testCase.Reps
+              testCase.Net.runSchedule();
```

Comment:
This is not informative.  runSchedule simply exits when empty.  Between measurements there should be a post to the signal:

```
net = sig.Net;
a = net.origin('a');
b = a.delay(0);
while (testcase.keepMeasuring)
  a.post(1)
  net.runSchedule
end
```

I'm not sure you can use `startMeasuring` within the while loop (you could check this) but including the post in the measurement is more informative than simply measuring a method call.

Reply:
ah right, just forgot to include the post